### DISCUSSION TITLE
Unable to spawn the Burger revenge from Menyoo

### DISCUSSION THREAD

**Discussion by Bradly7412** [YimMenu/YimMenu#2714] [Q&A]
In previous version of the menu i was able to spawn the burger model but as of: c87aea7f3cd581915ba0a1b3cddb9ab4fc9ef0a4
I am now unable to spawn it
[Burger.json](https://github.com/YimMenu/YimMenu/files/14274340/Burger.json)

**Comment by 99Anvar99** [comment]
> In previous version of the menu i was able to spawn the burger model but as of: [c87aea7](https://github.com/YimMenu/YimMenu/commit/c87aea7f3cd581915ba0a1b3cddb9ab4fc9ef0a4) I am now unable to spawn it [Burger.json](https://github.com/YimMenu/YimMenu/files/14274340/Burger.json)

I'm not sure if **XML vehicles/maps** accept .json file
correct me if it does.

**Reply by Bradly7412** [reply]
someone spawned the vehicle and i copied it as a persistent vehicle and was working. ever since the update it no longer works

**Reply by 99Anvar99** [reply]
Well, just tested without any code changes from latest commit in master and it works just fine i do not see an issue here.

Normal car test
![image](https://github.com/YimMenu/YimMenu/assets/60616540/e784f6f0-3b5f-46f8-8b7b-ca3730e7ce4b)

file that @Bradly7412 provided
![image](https://github.com/YimMenu/YimMenu/assets/60616540/81c14969-9ffb-4251-a2ae-0299ea62b2eb)

@gir489returns as you see apparently my code is working 😮

**Reply by Bradly7412** [reply]
the build that @gir489returns suggested does work for me. so i will mark this as the answer. Thanks for the help!
